elastic/e2e-testing
# Normalise APM traces to the same APM service when running in the CI

## What does this PR do?

Centralise the APM traces generated from the tests to use the configuration defined in the OpenTelemetry plugin when running in the CI.

Otherwise, it uses the vault configuration as used to be.

## Why is it important?

One single place to look at the APM traces.

## How to test this PR locally

Run the script `.ci/scripts/metricbeat-test.sh` then traces should be generated as usual in the `jenkins-stats`.

Run in the CI and traces should be generated in the `ci-stats`


## Follow ups

- Enable context propagation with the `TRACEPARENT` -> https://github.com/elastic/apm-pipeline-library/blob/3fbbaaa0fa17c9ae36bcf7e602cc7da11b98b7d2/vars/withOtelEnv.groovy#L46-L48
